Makefile

```diff
@@ -64,7 +64,7 @@ check:
 	docformatter --check $(DOCFORMATTER_OPTIONS) $(DOCFORMATTER_FILES)
 	black --check $(BLACK_FILES)
 	blackdoc --check $(BLACKDOC_OPTIONS) $(BLACK_FILES)
-	flake8 $(FLAKE8_FILES)
+	flakeheaven lint $(FLAKEHEAVEN_FILES)
```

Comment:
Can someone try and test this properly? I'm struggling to find a way to get this to fail for some reason, so need to check that it's actually doing something :sweat_smile:

Reply:
I installed a fresh conda virtual environment using the `environment.yml` in this branch and tried a simple command like `flakeheaven lint --max-doc-length 9 pygmt/src/basemap.py` which should fail, but no error was thrown. My guess is there's something broken in `flakeheaven`.